FittedSheets/SheetOptions.swift

```diff
@@ -67,7 +72,12 @@ public struct SheetOptions {
                 shrinkPresentingViewController: Bool? = nil,
                 useInlineMode: Bool? = nil,
                 horizontalPadding: CGFloat? = nil,
-                maxWidth: CGFloat? = nil) {
+                maxWidth: CGFloat? = nil,
```

Comment:
I don't think this whole initializer is even needed, but since that's how the library is, I'll match the pattern